cmd/authelia-scripts/cmd_docker.go

```diff
@@ -14,7 +14,7 @@ import (
 
 var arch string
 
-var supportedArch = []string{"amd64", "arm32v7", "arm64v8"}
+var supportedArch = []string{"amd64", "arm32v7", "arm64v8", "darwin"}
```

Comment:
```suggestion
var supportedArch = []string{"amd64", "arm32v7", "arm64v8", "darwin-amd64"}
```

Reply:
This would mean the Dockerfile also needs to be renamed. I went with only Darwin because I don’t believe Macs come in any other Architecture so we will only ever build amd64.

Reply:
I actually don't know whether they support other architectures. No big deal if you want to keep it like this here. I just proposed to be explicit.